src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -141,6 +143,12 @@ public void setLength(double goal) {
         SmartDashboard.putNumber("elevator/goal", Units.metersToInches(m_goal));
     }
 
+    public void overrideLength(double override){
```

Comment:
I don't understand why you added this routine. You are just adding extra methods to implement the algorithm that is the command. I think these routines should just be in the command; it calls getLength, computes new one, then calls setLength.